packages/ui-calendar/src/Calendar/index.tsx

```diff
@@ -145,12 +145,8 @@ class Calendar extends Component<Props> {
   }
 
   static defaultProps = {
-    children: null,
-    renderNextMonthButton: undefined,
-    renderPrevMonthButton: undefined,
-    renderNavigationLabel: undefined,
-    onRequestRenderNextMonth: () => {},
-    onRequestRenderPrevMonth: () => {},
+    onRequestRenderNextMonth: () => {}, // TODO check if its OK if this is removed
+    onRequestRenderPrevMonth: () => {}, // TODO check if its OK if this is removed
```

Comment:
I tried to be as safe as possible and did not remove anything that could have side effects. (but this is still something we should fix when converting these components to TS, because the user can set it intentionally to `undefined`

Reply:
Hm, that's true, we should look out for these.